Unity-Technologies/Graphics
# [11.x.x] Early camera color target assignment

### Purpose of this PR
This PR fixes a regression causing passthrough camera to not render. This is because the render target is fetched during AddRenderPasses, but the wrong target is setup during that call due to recent changes.

In the long run, we do not want the camera color target to be accessed outside of the ScriptableRenderPass callbacks, but to avoid breaking code we're allowing it for now. 

Currently, if there are any renderer features we create a camera color target no matter what. That means that we can assign the camera color target early in case it is needed during AddRenderPasses. 

---
### Testing status
Manually verified using the [3DCharacterUI](https://github.com/Unity-Technologies/UniversalRenderingExamples/tree/236090b6a9501ef2be2f35531bcfa519783e376a/Assets/_CompletedDemos/3DCharacterUI) from [UniversalRenderingExamples#236090b6a9501ef2be2f35531bcfa519783e376a](https://github.com/Unity-Technologies/UniversalRenderingExamples/tree/236090b6a9501ef2be2f35531bcfa519783e376a)
1. Open the 3DCharacterUI scene
2. Enter play mode
3. Click on one of the characters in the scene to spawn the character UI
4. Notice that the background is correctly blurred with the fix
